docs/rackhd/event_notification.rst

```diff
@@ -299,4 +303,50 @@ An example of multiple filters
 Web Hook APIs
 ^^^^^^^^^^^^^^^
 
-**TODO** (The APIs of web hooks developments are ongoing, this section wiil be updated when they are finished).
+
+**Create a new hook**
+
+
+.. code-block:: REST
+
+    POST /api/2.0/hooks
+    {
+        "url": "http://www.abc.com/def"
```

Comment:
As an example, it will be better if the payload contains more attributes, otherwise, you are recommending user to create the filter without `name` and `filters`.